### PR TITLE
Copy nvm install.sh script

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Copies install script from nvm https://github.com/nvm-sh/nvm/blob/master/install.sh

The actual PR with changes is here https://github.com/temporalio/cli/pull/50

## Why?

Working on `temporal` install.sh script. Created a separate PR for ease of review

## Checklist

1. Closes 

2. How was this tested:

not tested. Just a copy

3. Any docs updates needed?

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,502 @@
+#!/usr/bin/env bash
+
+# The MIT License (MIT)
+
+# Copyright (c) 2010 Tim Caswell
```
Comment (bergundy) [inline install.sh:5]: Probably want to update this

**Review by bergundy — COMMENTED** [review]
Do we want to put this script here?
It should be in a cdn.

**Review by bergundy — COMMENTED** [review]
Not sure why we need the entire nvm script here tbh

File: install.sh (hunk 1 above)
Comment (feedmeapples) [reply install.sh:5]: updated the license here https://github.com/temporalio/cli/pull/50

**Comment by feedmeapples** [comment]
> Not sure why we need the entire nvm script here tbh

i've created this PR first prior to https://github.com/temporalio/cli/pull/50 so it's easier (maybe) to review #50

**Pull request closed without merging**